jp2_pc/Source/Lib/Sys/RegInit.cpp

```diff
@@ -263,7 +263,7 @@ void SetAutoSave(BOOL b_init)
 //*********************************************************************************************
 BOOL bGetAutoSave()
 {
-	return GetRegValue(REG_KEY_AUTOSAVE, 1) != 0;
+	return GetRegValue(REG_KEY_AUTOSAVE, 0) != 0;
 }
```

Comment:
"Auto Save" means that a savefile is created whenever an assertion fails. Assertions fail `a lot`, so this is a gigantic performance drag.